NewHaven/test/TurnSequence.cpp

```diff
@@ -0,0 +1,105 @@
+//
+// Created by Muher on 2020-03-11.
+//
+
+#include <iostream>
+#include "Player.h"
+#include "Resources.h"
+
+using namespace std;
+
+int main(){
+    // *** This is the sequence of the currently active player ***
+
+    // Initiate decks
+    auto *bDeck = new BuildingDeck();
+    auto *hDeck = new HarvestDeck();
+
+    auto *player = new Player(); // Initiate player
+
+    // Start of the game, each player should have 6 buildings and 2 Harvest Tiles
+    for (int i = 0; i < 5; i ++)
+        (*player).drawBuilding(*bDeck->draw()); // Player draws a building from the deck
+    for (int i = 0; i < 2; i++)
+        (*player).drawHarvestTile(*hDeck->draw()); //Player draws a harvest tile from the deck
+
+    // This tile is kept FACE DOWN until played.
+    HarvestTile *shipmentTilePtr;
+    HarvestTile shipmentTile = *hDeck->draw();
+
+    shipmentTilePtr = &shipmentTile;
```

Comment:
Hand has a shipment member variable, which is a HarvestTile.

Reply:
Ah I didn't notice that, let me try to change things up according to that then! thanks